lib/ansible/modules/source_control/git.py

```diff
@@ -447,6 +469,17 @@ def clone(git_path, module, repo, dest, remote, depth, version, bare,
     if verify_commit:
         verify_commit_sign(git_path, module, dest, version)
 
+    if separate_git_dir_fallback:
+        separate_git_dir = os.path.abspath(separate_git_dir)
+        fallback_cmd = ['mv', os.path.join(dest, '.git'), separate_git_dir]
+        module.run_command(fallback_cmd, check_rc=True, cwd=dest)
+        try:
+            dot_git_file = open(os.path.join(dest, ".git"), "w")
+            dot_git_file.write("gitdir: %s" % separate_git_dir)
```

Comment:
Please be consistent while mixing up single and double quotes in strings. Always try to mimic the style of surrounding code, so that it would look like it all was written by the person.